plugins/jobs/src/js/jobsMenu.js

```diff
@@ -0,0 +1,32 @@
+import StringUtil from "#SRC/js/utils/StringUtil";
```

Comment:
🎨 no `.ts` 😭 (this file)

Reply:
Also, why does the name start with a small `j`? I think it should be `JobsMenu` 🤔

Reply:
please look at the jobsDelete answer.